guide/matrix-vs-telegram.md

```diff
@@ -31,16 +31,16 @@ Despite recent studies, Telegram's use of its own MTProto protocol remains a deb
+| **Registration** | Requires phone number or [purchasing so called "anonymous numbers" to be used in place of phone number](https://telegram.org/blog/ultimate-privacy-topics-2-0#sign-up-without-a-sim-card). | **Phone number is usually optional.** Depending on homeserver, **email is commonly required**. There is usually no human check after registration. |
 | **Username** | Users are identified by phone number or username (if set up, 5\~32 alphanumeric characters) to fellow users, and user IDs (around 9\~10 digits) for programming purposes. A display name can be added (no limit). | Users are identified by their MXID (eg. `@alice:example.com`), composed of the username (must be ASCII characters, upper case letters are not allowed) and the server name (not exceeding 255 characters when combined, including the introducing at symbol and the colon separating the parts). A display name can be optionally added (up to ~65200 bytes)[^2]. |
 | Avatar | Static or animated; limit unknown. Can be zoomed; the returned avatar has a maximum definition of 640x640. | **See "Attachments" for limits.** Can be zoomed (at least in Element/SchildiChat), in which case the avatar will be shown in the uploaded definition. Animated avatars are supported and will be rendered (at least in Element/SchildiChat). |
 | Profile description | **Supported**. | Will be supported using profile rooms. Not supported currently. |
-| Room-specific nicknames | Not supported, though group admins can talk on behalf of the whole group. | **Supported** (`/myroomnick`). Up to ~65200 bytes. [^2] |
+| Room-specific nicknames | Group admins can talk on behalf of the whole group or [have custom title shown next to their name](https://t.me/TelegramTips/68). | **Supported** (`/myroomnick`). Up to ~65200 bytes. [^2] |
 | Room-specific avatars | Not supported. | [**Supported**](../features/#attachments) (`/myroomavatar`). See "Attachments" for limits. |
 | 2FA | One-time token sent to another session. | Not required for login, but required (QR code, emoji verification, or Security Key) for viewing past encrypted messages. |
 | **Text messages** | Maximum 4096 characters. Supports Markdown. | **Up to ~65200 bytes (up to ~21270 bytes if a formatted message with plain text fallback sent).[^2] [Supports Markdown and HTML.](../features/#text)** |
-| Attachments | **Maximum 2 GB.** | Maximum 50~100 MB (for most homeservers; customizable if you run your own homeserver). |
-| Reactions | Very limited. Must be enabled by group admins in groups. | All unicode emotes and [text](../features/#reactions). |
+| Attachments | **Maximum 2 GB for free users and 4 GB for Premium users.** | Maximum 50~100 MB (for most homeservers; customizable if you run your own homeserver). |
+| Reactions | Free users can use limited set of reactions, [premium users can use custom emoji](https://t.me/premium/10). May be disabled by group admins in groups. | All unicode emotes and [text](../features/#reactions). |
```

Comment:
If you're gonna mention that group admins can disable emojis, then it bears reminding that it's possible to do that in Matrix too

Reply:
It would be inappropriate for me to make changes in Matrix features description as I do not have experience with Matrix. I left "Allow edits from maintainers" enabled in this PR to let you make adjustments you might think are necessary.


The goal of this particular change was to correct that currently reactions in group chats are enabled by default.